framework/BaseYii.php

```diff
@@ -528,11 +528,7 @@ public static function t($category, $message, $params = [], $language = null)
      */
     public static function configure($object, $properties)
     {
-        foreach ($properties as $name => $value) {
-            $object->$name = $value;
-        }
-
-        return $object;
+        return self::$container->configureObject($object, $properties);
```

Comment:
container could be unavailable if someone is not using it.

Reply:
Okey. It can be in opposite way. You can configure object using BaseYii or Yii method.
But I think it should be in one place (DRY)

Reply:
We can configure any/all of object 
or only `implements \yii\base\Configurable`?

Reply:
Any object.

Reply:
@bscheshirwork it configures any object.